bot.py

```diff
@@ -167,7 +167,9 @@ def adjust_msg(msg):
     try:
         for spirit in spirit_emoji_map:
             try:
-                msg = re.sub(f'^(.) {spirit} ', '\\1 ' + emoji_to_discord_map[spirit_emoji_map[spirit]] + ' ', msg)
+                # searches for keys from spirit_emoji_map and replaces with correct Discord emoji
+                # \\S+ matches the emoji representing the spirit; (.) does not successfully match ❤️
+                msg = re.sub(f'^(\\S+) {spirit} ', '\\1 ' + emoji_to_discord_map[spirit_emoji_map[spirit]] + ' ', msg)
```

Comment:
Alternatively, we could probably make a "character class" out of the `colors_to_emoji_map` list to use here, but that felt like a bit of overkill :shrug: